### PR TITLE
Add a task for kubelow.org/paddlejob to docs

### PR DESCRIPTION
**Pull request by tenzen-y**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
I added documentation for `kubeflow.org/paddlejob`.

Also, I moved all sample jobs to the `/static/examples/jobs` folder.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
Add documentation for kubeflow.org/paddlejob
```

### CODE REVIEW

File: site/static/examples/jobs/sample-paddlejob.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apiVersion: kubeflow.org/v1
+kind: PaddleJob
+metadata:
+  name: paddle-simple-cpu
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  paddleReplicaSpecs:
```
Comment (trasc) [inline site/static/examples/jobs/sample-paddlejob.yaml:9]:
```suggestion
spec:
  runPolicy:
    suspend: true
  paddleReplicaSpecs:
```
nit

**Review by trasc — COMMENTED** [review]
Two nits, otherwise LGTM

File: site/static/examples/jobs/sample-paddlejob.yaml (hunk 1 above)
Comment (tenzen-y) [reply site/static/examples/jobs/sample-paddlejob.yaml:9]: Previously, we decided not to specify the `suspend=true` and documented it as optional.

https://github.com/kubernetes-sigs/kueue/pull/778#discussion_r1195874307

File: site/content/en/docs/tasks/run_kubeflow_jobs/run_paddlejobs.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Run a PaddleJob"
+date: 2023-09-22
+weight: 6
+description: >
+  Run a Kueue scheduled PaddleJob
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Training Operator](https://github.com/kubeflow/training-operator) PaddleJobs.
```
[the remark this answers is not in the document]
Comment (tenzen-y) [reply site/content/en/docs/tasks/run_kubeflow_jobs/run_paddlejobs.md:9]: Let's update links for all kubeflowjobs.

[the remark this answers is not in the document]
Comment (tenzen-y) [reply site/content/en/docs/tasks/run_kubeflow_jobs/run_paddlejobs.md]: Done.

**Comment by tenzen-y** [comment]
@trasc Addressed.

**Review by trasc — COMMENTED** [review]
/lgtm

**Pull request merged**